doc/api/source_map.md

```diff
@@ -0,0 +1,85 @@
+# Source Map
+
+<!--introduced_in=REPLACEME-->
+
+> Stability: 1 - Experimental
+
+<!--name=source_map-->
+
+Exposes an API for interacting with Node.js' internal cache of source maps.
```

Comment:
Utter micro-nit but it might be easiest for readers if we don't use the possessive in this instance:

```suggestion
Exposes an API for interacting with the internal cache of source maps.
```

...or...

```suggestion
Exposes an API for interacting with the Node.js internal cache of source maps.
```

Or even:

```suggestion
Exposes an API for interacting with the cache of source maps.
```